examples/slack-connector/src/main/resources/application.properties

```diff
@@ -0,0 +1,47 @@
+########################################################################
+#Properties used by channel connector connector framework.
+########################################################################
+
+#The base URL for Chatbot Runtime.
+sfdc.einstein.bots.runtime-url=${BOT_RUNTIME_URL:https://runtime-api-na-west.stg.chatbots.sfdc.sh}
+
+#The name of the integration that will be used. This should match the Integration Name used
+#when adding connected app API connection to Bot.
+sfdc.einstein.bots.integration-name=${BOT_INTEGRATION_NAME:sfdc.chatbot.api}
+
+#OAuth Properties
+#Path to the private key needed to get OAuth Tokens.
+sfdc.einstein.bots.oauth.private-key-file=${OAUTH_PRIVATE_KEY_FILE_PATH:/app/target/classes/private_key.der}
+
+#The URL to hit to obtain an OAuth Token
+sfdc.einstein.bots.oauth.login-endpoint=${LOGIN_END_POINT:https://login.test1.pc-rnd.salesforce.com/}
+
+#ID of the Connected app you setup for API Connection.
+sfdc.einstein.bots.oauth.connected-app-id=${CONNECTED_APP_ID:3MVG9l3R9F9nGODSUSsBSTRIINiGzUl5y6Fi.G6JMNGh54OFHBADNVvyx5XegzsVkKUss0zXxR78rb0FSYIxT}
+
+#Secret of the Connected app you setup for API Connection.
+sfdc.einstein.bots.oauth.connected-app-secret=${CONNECTED_APP_SECRET:78BF17D1787B59B6D0F598EB0BA406587ADCB379B45A36DAD59E55A9D3668F62}
```

Comment:
cannot have secrets in open source.

Reply:
None of them are real secrets - It's listed in the README that the secrets are just samples so they can co-relate the size of each of them. I'll remove them all though.

Reply:
Removed